site/docs/governance.md

```diff
@@ -64,6 +64,7 @@ The rules require that a community member registering a negative vote must inclu
 | Non-breaking function introductions                                                                       | 1 (not including proposer) | Committers     | Github PR       |
 | Non-breaking extension additions & non-format code modifications                                          | 1 (not including proposer) | Committers     | Github PR       |
 | Changes (non-breaking or breaking) to a Substrait *library* (i.e. substrait-java, substrait-validator)    | 1 (not including proposer) | Committers     | Github PR       |
+| Changes to a Substrait `-contrib` repository                                                              | 1 (not including proposer) | Contributers   | Github PR       |
```

Comment:
Let's add an additional line for adding a contributor to a -contrib repo.

Reply:
Done in latest commit.